content/index.html

```diff
@@ -67,7 +67,11 @@ <h2 class="faq"><i class="fas fa-comments"></i></h2>
     </li>
     <li>
         <p class="heading">Do you have plans for supporting iOS apps?</p> 
-        <p>Yes, in the long run, we'd like to be able to run iOS apps on ARM devices (like most Android phones). A significant challenge here would be to write our own implementation of UIKit. Come talk to us if you're interested in working on this!</p>
+        <p>Yes, in the long run, we'd like to be able to run iOS apps on ARM devices (like most Android phones). Come talk to us if you're interested in working on this! In the future, support for tvOS, watchOS, and visionOS software is planned too.</p>
+    </li>
+    <li>
+        <p class="heading">Which arcetextures of are supported?</p> 
+        <p>Currently, x64 and x86 (64 and 32-bit Intel) macOS software are supported, with ARM64 support being in the works. In the future, we will support ppc (PowerPC) and ppc64 (PowerPC G5) for macOS apps, as well as ARM32 for older iOS apps and a few other archetexures for running NextSTEP software. A Rosetta-like system based on qemu-user-static will also be implemented.</p>
```

Comment:
> a few other archetexures for running NextSTEP software.
1. ~~arcetextures~~ architectures
2. Did anyone in the Darling team agree to this?

Reply:
It's been discussed in certain issues (e.g. darlinghq/darling#252), but we certainly haven't made any promises (which this statement is committing us to adding PPC support).

Reply:
x64 -> x86_64; "PowerPC G5" is not an architecture. And really I don't think the frontpage should mention anything but the Intel/ARM ones. We could perhaps say "supporting more architectures would be possible in the future", without any specifics.